app/uk/gov/hmrc/cardpaymentfrontend/services/CardPaymentService.scala

```diff
@@ -87,10 +87,11 @@ class CardPaymentService @Inject() (
   }
 
   def finishPayment(transactionReference: String, journeyId: String)(implicit headerCarrier: HeaderCarrier): Future[Option[CardPaymentResult]] = {
+    logger.info("TransactionReference: " + transactionReference) //todo take me out before go live
```

Comment:
for any TODO's it would be prudent to create tickets for work that needs to be done if you haven't already, otherwise we may forget about it. Do you definitely need this?